File: src/main/webapp/app/domain/hospital.ts
import {WaitingList} from 'app/domain/waiting-list';
import {Location} from 'app/domain/location';

export class Hospital {
  constructor(name: string, waitingList: WaitingList[], waitTime: number) {
    this.name = name;
    this.waitingList = waitingList;
    this.waitTime = waitTime;
  }

  id: bigint;
  name: string;
  waitingList: WaitingList[];
  location: Location;
  waitTime: number;
}

File: src/main/java/com/paloit/zombieapocalypse/domain/Hospital.java
package com.paloit.zombieapocalypse.domain;

import lombok.*;

import java.io.Serializable;

@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Hospital implements Serializable {

  private static final long serialVersionUID = -823230006552057599L;

  @NonNull
  private Long id;
  private String name;
  private WaitingList[] waitingLists;
  private Location location;
}
